Crear resumen de la noticia

Morban15 opened the pull request:
Hemos añadido teaser.html a la carpeta News.

### Issues

_¿Qué se está resolviendo?_
Creando el archivo html para el resumen de la noticia.
#17 

### Todos

_¿Qué queda por hacer?_
El archivo css, hacerlo en los dos idiomas, revisar, terminar y merged.

### Áreas impactadas

_¿Qué páginas o componentes están afectados?_
teaser.html

Morban15 committed:
Añadiendo el archivo HTML a la carpeta News

Hemos añadido teaser.html a la carpeta News.

Morban15 committed:
Añadiendo el archivo HTML a la carpeta News

Hemos añadido teaser.html a la carpeta News.

Morban15 committed:
Haciendo cambios e indentando en html

Modificamos el archivo teaser.html

Morban15 committed:
Haciendo cambios e indentando en html

Modificamos el archivo teaser.html

Morban15 committed:
Cambiando el nombre de la clase que esta en el tag p

Hemos cambiado el nombre de la clase de "teaser-lines" a
"teaser-description".

Morban15 committed:
Cambiando el nombre de la clase que esta en el tag p

Hemos cambiado el nombre de la clase de "teaser-lines" a
"teaser-description".

Morban15 committed:
Agregando el yml en ingles

Hemos agregado el objeto teaser en news.yml en ingles.

Morban15 committed:
Agregando el yml en ingles

Hemos agregado el objeto teaser en news.yml en ingles.

Morban15 committed:
Agregando el yml en español

Hemos agregado el objeto teaser en news en español.

Morban15 committed:
Agregando el yml en español

Hemos agregado el objeto teaser en news en español.

Morban15 committed:
Agregando las propiedades de objeto teaser al archivo de HTML

Agregamos las propiedades del objeto teaser en teaser.html.
(aun no lo hemos probado)

Morban15 committed:
Agregando las propiedades de objeto teaser al archivo de HTML

Agregamos las propiedades del objeto teaser en teaser.html.
(aun no lo hemos probado)

Morban15 committed:
Agregando el teaser a la pagina news

Hemos agregado el archivo teaser.html a news.
(no hemos probado)

Morban15 committed:
Agregando el teaser a la pagina news

Hemos agregado el archivo teaser.html a news.
(no hemos probado)

Morban15 committed:
Cambiando el nombre de la carpeta news a teaser_news

Teniamos dos carpetas llamadas news y cambiamos una de ellas a teaser_news

Morban15 committed:
Cambiando el nombre de la carpeta news a teaser_news

Teniamos dos carpetas llamadas news y cambiamos una de ellas a teaser_news

Morban15 committed:
Cambiando el nombre del archivo programs a programas

Cambiamos de programs a programas porque existian dos archivos programs y uno
de ellos debe estar en español.

Morban15 committed:
Cambiando el nombre del archivo programs a programas

Cambiamos de programs a programas porque existian dos archivos programs y uno
de ellos debe estar en español.

Morban15 committed:
Cambiando el nombre del archivo becas a scholarships

Cambiamos el nombre del archivo becas porque exisian dos archivos llamados becas
y uno de ellos debe estar en español.

Morban15 committed:
Cambiando el nombre del archivo becas a scholarships

Cambiamos el nombre del archivo becas porque exisian dos archivos llamados becas
y uno de ellos debe estar en español.

Morban15 committed:
Cambiando el nombre del archivo donate

Cambiamos el nombre del archivo donate a donar

Morban15 committed:
Cambiando el nombre del archivo donate

Cambiamos el nombre del archivo donate a donar

Morban15 committed:
Resolviendo problemas en i18n.js

Cambiando el nombre de algunos objetos en el archivo js.

Morban15 committed:
Resolviendo problemas en i18n.js

Cambiando el nombre de algunos objetos en el archivo js.

Morban15 committed:
Fixing gulp tasks

Looking for a taskt that was renamed
Add copy-image task to default

Morban15 committed:
Fixing gulp tasks

Looking for a taskt that was renamed
Add copy-image task to default

Morban15 committed:
Haciendo algunos cambios para la pagina news

Hemos hecho algunas odicficaciones para probar y mejorar news

Morban15 committed:
Haciendo algunos cambios para la pagina news

Hemos hecho algunas odicficaciones para probar y mejorar news

Morban15 committed:
Creando el archivo teaser.css para teaser_news

Hemos creado el archivo de css para aplicarle el RWD

Morban15 committed:
Creando el archivo teaser.css para teaser_news

Hemos creado el archivo de css para aplicarle el RWD

misscs commented:
😃 🥇 Este PR es **fenomenal!** Sus mensajes de commit son 💯. Y tambien se puede ver que ustedes estan tomando el tiempo para completar la tarea diligentemente. Estoy impresionado. 

También es una buena idea abrir los PRs temprano como hicieron. 👍

Morban15 committed:
Merge branch 'teaser' of https://github.com/cosola-macile/macile.org into teaser

Morban15 committed:
Importando el archivo de CSS a site.css

Hemos importado el teaser.css a site.css.

Morban15 committed:
WIP

Mejorando algunas cositas para probar.

Morban15 committed:
WIP

Estamos haciendo algunos cambios

misscs commented:
@Morban15 Esto esta lista para revisar?

Morban15 committed:
Trabajo terminado para el resumen de la noticia

Hicimos algunos cambios para llegar al resultado deseado.
El archivo de teaser_news esta en la pagina News.

Morban15 commented:
@misscs ya, es el ultimo

misscs commented on app/modules/page/site_base.css:
```diff
@@ -52,7 +52,7 @@ body {
 }
 
-img{
+img {
```
👍

misscs commented on app/modules/teaser_news/teaser.css:
```diff
@@ -0,0 +1,21 @@
+.teaser-block {
+  display: flex;
+  background-color: pink;
+  width: 50vh;
```
👍 Genial usando las nuevas unidades de longitud.

misscs commented on app/modules/teaser_news/teaser.css:
```diff
@@ -0,0 +1,21 @@
+.teaser-block {
+  display: flex;
+  background-color: pink;
```
Creo que podemos remover el rosado :)

misscs commented on app/modules/teaser_news/teaser.html:
```diff
@@ -0,0 +1,10 @@
+<article class="teaser-block">
+    <figure>
```
Me imagino que eventualmente tendremos que añadir estilos al imagen. Puedes darle una clase?

misscs commented on app/modules/teaser_news/teaser.html:
```diff
@@ -0,0 +1,10 @@
+      <img src="{{ teaser.image }}" alt="{{ teaser.alt }}" />
+    </figure>
+    <div class="teaser_content">
+        <a href="{{ teaser.href_path }}"><h2>{{ teaser.title }}</h2></a>
```
Es pura preferencia, pero si yo solamente tengo un elemento que necesita un link, a mi me gusta poner el link a dentro:

```
<h2><a href=""></a></h2>
```

Poner el link alrededor del elemento es perfectamente válido. Cuando yo empecé a programar no se podia hacerlo 😆 

No la cambias! Para ahora esta bien. Dejemos que el diseño decida :)

misscs commented on config/i18n.js:
```diff
@@ -23,8 +24,9 @@ export const i18n = {
       {view: `programs`, path: `/programas`},
       {view: `about`, path: `/sobre`},
-      {view: `becas`, path: `/becas`},
+      {view: `scholarships`, path: `/becas`},
```
Buena corrección!!! 🐛 ✅

misscs reviewed (changes requested):
💯 💯 💯  Por tomar la iniciativa para corregir errores!! 

Quiero trabajar contigo mañana para para mostrarte algunas otras cosas que podemos hacer con flexbox para mejorar el imagen. 

También necesitamos añadir un resumen sin un foto. Voy a dar una clases el lunes sobre conditionals y loops en las templanzas. Para ahora, puedes crear outro objecto de contenido en yml. Y en `news/index.html` utilizar:

```
{% set teaser = teaser %}
{% include "teaser_news/teaser.html"%}

{% set teaser = teaser_2 %}
{% include "teaser_news/teaser.html"%}
```

En `teaser_news/teaser.html` tenemos que envolver el imagen en un conditional 


```
    {% if teaser.image %}
      <figure>
        <img src="{{ teaser.image }}" alt="{{ teaser.alt }}" />
      </figure>
    {% endif %}
```

El lunes también hablaré más sobre flexbox

Morban15 committed:
Agregando el objeto teaser_2 a noticias.yml

Agregamos el objeto teaser_2 para la traducción en español.

Morban15 committed:
Agregando la parte en inglés a teaser_2

Agregamos el objeto teaser_2 para la traducción en inglés.

Morban15 committed:
Creando otro objeto de contenido en yml

Creamos un objeto teaser_2

Morban15 committed:
Arreglamos alt en teaser

Arreglamos alt en teaser.html

Morban15 committed:
Envolviendo la imagen en una condicional

Agregando una condicional a la imagen en teaser.html

Morban15 committed:
Removiendo el rosado de teaser.css

Eliminamos el background rosado en el css

Morban15 committed:
Agregando una clase a figure en teaser.html

Agregando clase a figure para poder hacer cambios en la imagen.

Morban15 committed:
Agregando cambios en teaser.css

Corrigiendo RWD

Morban15 committed:
Arreglando nombres de las clases en teaser.html

Para que no choquen.

misscs committed:
Merge branch 'master' into teaser

Conflicts:
	app/modules/page/site_base.css
	app/views/news/index.html
	config/i18n.js
	content/en/news.yml
	gulpfile.babel.js

misscs reviewed (approved):
MERGE!! 🙌
